pkg/appfile/config/local.go

```diff
@@ -19,9 +19,14 @@ package config
 import (
 	"bufio"
 	"bytes"
+	"context"
 
+	"github.com/oam-dev/kubevela/pkg/utils/common"
 	"github.com/oam-dev/kubevela/pkg/utils/config"
 	env2 "github.com/oam-dev/kubevela/pkg/utils/env"
+
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	cli_conf "sigs.k8s.io/controller-runtime/pkg/client/config"
```

Comment:
```suggestion
	config2 "sigs.k8s.io/controller-runtime/pkg/client/config"
```

underline is not suggested